src/index.css

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Lato');
```

Comment:
- [ ] It would be better if you don't commit files that your project will not use. Kindly add this file to .gitignore or delete. 😄 👍